pymodbus/framer/rtu_framer.py

```diff
@@ -89,14 +89,9 @@ def checkFrame(self):
             self.populateHeader()
             frame_size = self._header['len']
             data = self._buffer[:frame_size - 2]
-            crc = self._buffer[frame_size - 2:frame_size]
```

Comment:
`crc` is already extracted from message and can be accessed as `self._header['len']`, no need to extract it again